packages/lending/src/protocols/aave-v3/configs.ts

```diff
@@ -277,6 +337,47 @@ export const configs: Config[] = [
       },
     ],
   },
+  // https://github.com/bgd-labs/aave-address-book/blob/main/src/AaveV3Base.sol
+  {
+    chainId: common.ChainId.base,
+    contractMap: {
+      Pool: '0xA238Dd80C259a72e81d7e4664a9801593F98d1c5',
+      PoolDataProvider: '0x2d8A3C5677189723C4cB8873CfC9C8976FDF38Ac',
+      AaveOracle: '0x2Cc0Fc26eD4563A5ce5e8bdcfe1A2878676Ae156',
+    },
+    reserves: [
+      {
+        asset: baseTokens.ETH,
+        aToken: baseTokens.aBasWETH,
+        used: { deposit: true, withdraw: true, borrow: true, repay: true, flashLoan: true },
```

Comment:
native token can not flash loan
can ref [line 95](https://github.com/dinngo/protocolink-js-sdk/blob/master/packages/lending/src/protocols/aave-v3/configs.ts#L95)